### ISSUE TITLE
no table or plot buttons in output

### ISSUE THREAD

**Issue by turbomam** [polynote/polynote#616]
I'm not currently getting the table and plot buttons in the output area.

- Ubuntu 18.04 running under WSL
- openjdk version "11.0.4"
- spark-2.4.4-bin-hadoop2.7.tgz
- polynote-dist.tar.gz from release 0.2.11
- browsing from Chrome 78.0.3904.87 on the Windows host (via localhost)

I have tried a few other PolyNote releases in the past, directly under Ubuntu 18.10, and I was able to create plots with the buttons, but I can't recreate that now.

![image](https://user-images.githubusercontent.com/25185273/68075739-47947c00-fd82-11e9-91c0-f30445894b07.png)

**Comment by jeremyrsmith** [comment]
If you run the cell again (a second time), do you see the schema display and plot buttons?

I've noticed that sometimes the first cell run just doesn't get the right `ReprsOf`. Some weird issue with implicit resolution/compiler state. Will dig into it more, but can you see if re-running the cell helps?

**Comment by turbomam** [comment]
No, they don't appear on a subsequent run.

I am getting various error messages, and I'm not sure how "serious" or relevant they are.

One starts:

```
[blaze-selector-3] INFO org.http4s.blaze.channel.nio1.NIO1SocketServerGroup - Accepted connection from /127.0.0.1:54814
[ERROR] Error creating result reprs
      : java.lang.NoClassDefFoundError: sun/misc/Cleaner
        polynote.runtime.StreamingDataRepr$.fromHandle(ValueRepr.scala:264)
```

**Comment by jeremyrsmith** [comment]
Well that's new! I guess the `sun.misc.Cleaner` API has become inaccessible in newer JDK versions. Spark has dealt with a similar issue: https://issues.apache.org/jira/browse/SPARK-24421

Will try accessing `Cleaner` through reflection instead, and see if that helps. Thanks for the issue report!

**Comment by jeremyrsmith** [comment]
We use `Cleaner` to ensure that an unfinished streaming iterator gets destroyed when its handle is no longer referenced. Maybe there's a better way to do that without relying on `Cleaner`. Should also investigate that some more.

**Comment by jeremyrsmith** [comment]
@turboman were you previously using JDK8? You mentioned that previously you were able to get the schema display and associated plotting capability – if you were already using the same JDK it would be more difficult to explain why it stopped working...

**Comment by turbomam** [comment]
I set up a fresh Ubuntu 18 EC2 instance on AWS with openjdk 8 and polynote-dist.tar.gz from release 0.2.11.

Still no schema browser or plot tool icons.

Yes, I did get those icons in the past. I habitually use default-jdk on Ubuntu, so I think it was Java 11 last time I saw them. I think I set that system up last weekend, so that may give you some rough idea of which build it was. Sorry I don't take better notes... it was working great, but I was paying for AWS.

**Comment by turbomam** [comment]
Icons are present with 0.2.7 and openjdk 8

**Comment by jeremyrsmith** [comment]
@turboman when it doesn't work in JDK 8, is it the same error? (About `java.lang.NoClassDefFoundError: sun/misc/Cleaner`)

**Comment by turbomam** [comment]
@jeremyrsmith ... I don't recall.  I can try to recreate that tomorrow. 

As you suspected, the buttons are present with every PolyNote build I have tried, on multiple platforms, as long as I stick to openjdk 8. When I switch to openjdk 11, the buttons go away, and then don't immediately reappear when I switch to back to openjdk 8. I thought I was resetting my JAVA_HOME each time, but I don't see that consistently in my history.

```BASH
sudo apt-get remove default-j* openjdk* java-common
sudo apt-get install openjdk-8-jdk
./polynote/polynote
```

I have inconsistently tried other things like rebooting and browser-cache clearing too

**Comment by jeremyrsmith** [comment]
Should be solved by #631

**Comment by turbomam** [comment]
I can confirm that `polynote-dist-2.12.tar.gz` v. 0.2.13 is working nicely with openjdk version "11.0.4" on Ubuntu 18.04.
